File: Python/card.py
from payment import Payment

class Card(Payment):
    card_type : str
    expiration_date: int
    cvv: str
    
    def __init__(self, id, card_type, expiration_date, cvv):
        super().__init__(id)
        self.card_type = card_type
        self.expiration_date = expiration_date
        self.cvv = cvv

File: Java/Card.java
public class Card extends Payment{
    
    String cardType;
    Integer expirationDate;
    String cvv;

    public Card(Integer id, String cardType, Integer expirationDate, String cvv) {
        super(id);

        this.cardType = cardType;
        this.expirationDate = expirationDate;
        this.cvv = cvv;

        this.printDataPayment();
    }

}